scripts/eurostat/health_determinants/common/denominator_mcf_generator.py

```diff
@@ -0,0 +1,140 @@
+import re
```

Comment:
what is this script used for? much of the code looks duplicated in process.py - are they both needed?